vispy/visuals/polygon.py

```diff
@@ -38,17 +36,34 @@ class PolygonVisual(Visual):
     """
     def __init__(self, pos=None, color='black',
                  border_color=None, border_width=1, **kwargs):
-        super(PolygonVisual, self).__init__(**kwargs)
-
-        self.mesh = MeshVisual()
```

Comment:
@campagnola I'd prefer to create a separate ```@property``` to access them so that it makes it _explicit_ that ```mesh``` and ```border``` can be accessed from the outside (and I think it's easier to document a specific ```@property``` than documenting raw class methods?

So, the class would still have ```self._mesh```, but there would be a

```
    @property
    def mesh(self):
        """ The mesh of the polygon.
        """
        return self._mesh

    @mesh.setter
    def mesh(self, mesh):
        self._mesh = mesh
        self._update()
```

It that an anti-pattern? Or is that okay? It pleases my C++ side, but I'm not sure if that's pythonic :)